### ISSUE TITLE
action inside list block

### ISSUE THREAD

**Issue by 0000marcell** [cibernox/ember-power-select#1069]
is there a way to insert a button with an action inside the list and have this action called without selecting the item. 
something like this
```
{#power-select-multiple  
  selected=selectedTags
  searchField="name"
  selectedItemComponent="selected-item-tag"
  onkeydown=(action "handleKeydown")
  options=tags onchange=(action "selectTag") 
  as |item|}}
  <div class="flex-row inline-space-between">
    {{item.name}}
    <a {{action "removeTag" item bubbles=false}}>x</a>
  </div>
{{/power-select-multiple}}
```

i've tried with bubbles=false and capturing the event and calling even.stopPropagation(), 
but the selection is always called first 😢

**Comment by cibernox** [comment]
Yes, there is. What is tripping you is that the options are not selected on click, but on `mousedown` (seems weird, but this is how it should be to behave like native selects do).

What I recommend is to prevent the propagation of the mouseup but still file the button's action on click:

```
{#power-select-multiple  
  selected=selectedTags
  searchField="name"
  selectedItemComponent="selected-item-tag"
  onkeydown=(action "handleKeydown")
  options=tags onchange=(action "selectTag") 
  as |item|}}
  <div class="flex-row inline-space-between">
    {{item.name}}
    <a onmouseup={{action "stopPropagation"}} onclick={{action "removeTag"}}>x</a>
  </div>
{{/power-select-multiple}}
```

**Comment by 0000marcell** [comment]
Thx a lot Miguel, what do you think about adding this info to the docs? i think its a pretty common use case and is hard to figure out alone, at least it was for me, i'm probably will have to come back here to remember what i need to do, maybe adding this info to "the list" section of the docs, i could open a PR if you think its a good idea.

**Comment by cibernox** [comment]
Sure, although I'm not sure where to put it. Where would you have looked for this information?

**Comment by 0000marcell** [comment]
I think "basic customization > the list"   is the best place to put it, a new topic called "prevent click propagation" what do you think ?

**Comment by cibernox** [comment]
Could work, yes. Do you want to make a PR? The documentation is just the dummy app inside this repo. It's pretty straightforward.
